Foundation/Bundle.swift

```diff
@@ -53,8 +53,16 @@ open class Bundle: NSObject {
         self.init(path: url.path)
     }
     
-    public init(for aClass: AnyClass) { NSUnimplemented() }
-    
+#if os(Android)
```

Comment:
Can you explain your intent here?

Reply:
This is for compatibility for Open Source code notably Alamofire to run (almost) without modification. It uses the main bundle to try to read the Info.plist to get the app version. Non of this works but with this change at least it doesn’t crash.

Reply:
The main bundle is fetched via `Bundle.main`, though. How is this path exercised by that?

Reply:
Alamofire calls Bundle(for: aClass)